socket/packet/server_list_request.go

```diff
@@ -0,0 +1,20 @@
+package packet
+
+import (
+	"github.com/sandertv/gophertunnel/minecraft/protocol"
+)
+
+// ServerListRequest is sent by the client to request list of all the servers
+// connected to portal proxy (including offline servers).
```

Comment:
Make doc comments maximum of 110 characters per line instead of how it looks best visually

Reply:
it's 94 chars length now